src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/external-references/ExternalReferences.scss

```diff
@@ -0,0 +1,81 @@
+@import 'src/styles/common';
+
+.titleWithSeparator {
```

Comment:
This and some other classes in this file are complete copy of the classes in Overview.scss. Doesn't this suggest that either all sidebar css should go in a single file or that there should be a file with common classes?

Reply:
There are only 2 classes that are common and I don't think it will make a big difference putting it all in a single file.